man/man8/zfs-list.8

```diff
@@ -63,12 +63,15 @@ If specified, you can list property information by the absolute pathname or the
 relative pathname.
 By default, all file systems and volumes are displayed.
 Snapshots are displayed if the
-.Sy listsnaps
-property is
+.Sy listsnapshots
+pool property is
 .Sy on
 .Po the default is
 .Sy off
-.Pc .
+.Pc ,
+or if the
+.Fl t Sy snapshot
+option is specified.
```

Comment:
[nit] I usually do `zfs list -t all` when I want to see snapshots (I also want to see everything else).  How about saying `if the -t snapshot or -t all options are specified`

Reply:
Sure, updated.